runtime/ops/io.rs

```diff
@@ -60,15 +59,15 @@ static STDERR_HANDLE: Lazy<StdFile> =
 // TODO(ry) It should be possible to close stdout.
 #[cfg(windows)]
 static STDIN_HANDLE: Lazy<StdFile> = Lazy::new(|| unsafe {
-  StdFile::from_raw_handle(GetStdHandle(winbase::STD_INPUT_HANDLE))
+  StdFile::from_raw_handle(std::io::stdin().as_raw_handle())
 });
 #[cfg(windows)]
 static STDOUT_HANDLE: Lazy<StdFile> = Lazy::new(|| unsafe {
-  StdFile::from_raw_handle(GetStdHandle(winbase::STD_OUTPUT_HANDLE))
+  StdFile::from_raw_handle(std::io::stdout().as_raw_handle())
```

Comment:
I think this is probably wrong due to this comment...

https://github.com/denoland/deno/blob/d4ad2b809c62b59ee4c3767bf1ff096b30886bdb/runtime/ops/io.rs#L53-L54

...but I'm not sure why doing this change fixes the issue.